### ISSUE TITLE
HERE only terrainDay seems to work

### ISSUE THREAD

**Issue by paazmaya** [leaflet-extras/leaflet-providers#130]
I am using a simple map as shown below, with the API keys omitted:

```js

    var map = L.map('map').setView([-29.95315222951503, 24.57483478], 6);

    /*
     HERE.normalGreyDay
     HERE.satelliteNoLabelsDay
     HERE.satelliteYesLabelsDay
     HERE.terrainDay
    */
    L.tileLayer.provider('HERE.terrainDay', {
      app_id: '***',
      app_code: '***'
    }).addTo(map);
```
Libraries are loaded from:

```html
  <script src="//cdnjs.cloudflare.com/ajax/libs/leaflet/0.7.3/leaflet.js"></script>
  <script src="//cdnjs.cloudflare.com/ajax/libs/leaflet-providers/1.0.7/leaflet-providers.min.js"></script>
```
However when changing the map variant to any other than the one used in the example from `README.md`, namely `'HERE.terrainDay'`, the following message appears to console:

```
uncaught exception: No such variant of HERE (normalGreyDay)
````

I assuming that internally the "Map Tile API" is being used, is that correct?
https://developer.here.com/documentation/versions

**Comment by paazmaya** [comment]
Well, here they seem to be defined
https://github.com/leaflet-extras/leaflet-providers/blob/master/leaflet-providers.js#L355
which would mean that `README.md` is out of date.

**Comment by jieter** [comment]
The README is indeed not correct, the variant name is `normalDayGrey` and not `normalGreyDay`.

Thanks for reporting!

**Comment by jieter** [comment]
released 1.0.15
